README.md

```diff
@@ -26,4 +26,12 @@ func param_test_function(param1 : String):
 
 The "quit"/"exit" command is implemented by default.
 
+By default the console pauses the tree (to give the user time to enter commands). If this is undesirable behaviour to you, you can change that behaviour by setting the `pause_enabled` variable accordingly.
```

Comment:
Needs to be update for new default pause behavior

Reply:
Oops, you're absolutely right!